cmd/main.go

```diff
@@ -86,6 +87,10 @@ func main() {
 		&config.DisableExporterMetrics, "disable-exporter-metrics", true, "Excludes additional process or go runtime related metrics (i.e process_*, go_*). Default is true.",
 	)
 
+	cmd.PersistentFlags().StringVar(
+		&config.IgnoreBlockDevicesRegex, "ignore-block-devices-regex", "", "Ignore the Stroage devices by specifying the matching Regular Expression",
```

Comment:
```suggestion
		&config.IgnoreBlockDevicesRegex, "ignore-block-devices-regex", "", "Ignore the storage devices by specifying the matching regular expression",
```
typo /stroage/storage/

Reply:
Do we have a default behaviour for this flag. ?

Reply:
If we don't pass this flag or Pass an empty string, This flag will be ignored and it will allow all the block devices